package.json

```diff
@@ -24,7 +24,7 @@
     "update:cache": "yarn workspaces foreach install --check-cache",
     "update:dedupe": "yarn workspaces foreach dedupe",
     "preview": "run-s -l preview:antora preview:hugo",
-    "preview:antora": "antora --clean --fetch antora-playbook.yml",
+    "preview:antora": "antora --clean --fetch antora-playbook-production.yml",
```

Comment:
Perhaps remove `preview:antora` altogether and replace `preview:antora` in `preview` script with `build:antora` instead.

```suggestion
```